pandas/core/arrays/timedeltas.py

```diff
@@ -355,6 +355,31 @@ def to_pytimedelta(self):
         """
         return tslibs.ints_to_pytimedelta(self.asi8)
 
+    def to_timedelta64(self):
+        """
+        Return numpy array with timedelta64[ns] dtype
+
+        Returns
+        -------
+        ndarray[timedelta64[ns]]
+
+        Notes
+        -----
+        This returns a view on self, not a copy.
+
+        See also
+        --------
+        Timedelta.to_timedelta64
+        """
+        return self.asi8.view('m8[ns]')
```

Comment:
you are going a little bit in circles here, as `asi8` itself is already `self.values.view('i8')` ?

Reply:
I find `asi8` much more explicit than alues (BTW as I mentioned to jreback the "v" key on my keyboard is sticky; I'm going to stop copy-pasting and let you guys infer the missing letter for a while).